lnbits/core/templates/core/wallet.html

```diff
@@ -660,7 +660,7 @@ <h6 v-else class="q-my-none">
               <p v-if="parse.lnurlpay.fixed" class="q-my-none text-h6">
                 <b>{{ parse.lnurlpay.domain }}</b> is requesting {{
                 parse.lnurlpay.maxSendable | msatoshiFormat }}
-                {{LNBITS_DENOMINATION}}
+                <span v-text="parse.data.unit"></span>
```

Comment:
Revert this line change

When I change the currency, the dialog claims the LNaddress requests amount in that currency (between 1 and 1,000,000 CZK) which is not the case. The title of the header window should be still kept in sats.

<img width="506" alt="Screenshot 2023-11-30 at 9 52 06" src="https://github.com/lnbits/lnbits/assets/42201/8efddb37-94a4-42bb-a980-b3da52ba68f4">